inputs/wysiwyg.vue

```diff
@@ -759,15 +830,17 @@
 
       editor.on('selection-change', (range) => {
         if (range) {
+          tooLong && tooLong.add(maxLength, editor.getLength());
           this.onFocus();
         } else {
+          tooLong && tooLong.remove();
           this.onBlur();
         }
       });
 
-      editor.on('text-change', function onTextChange() {
+      editor.on('text-change', function onTextChange({}, {}, source) {
```

Comment:
what are the first two arguments here?

Reply:
This is equivelnat to calling `function(_, arg) {}` in some languages when you only care about the 2nd arg.

Reply:
hm! the more you know!